src/compiler/emitter.ts

```diff
@@ -148,10 +148,10 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
             let emit = emitNodeWithoutSourceMap;
 
             /** Called just before starting emit of a node */
-            let emitStart = function (node: Node) { };
+            let emitStart = function(node: Node) { };

```

Comment:
the default formatting options for VS is to add the space. can we keep it this way.